crates/router/src/types/api/configs.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Clone, serde::Deserialize)]
+pub struct ConfigKeyCreate {
+    pub key: String,
+    pub value: String,
+}
```

Comment:
Would be better if you could rename this as `Config` (or something similar), excluding the `Create`/`Retrieve` operation name. From the looks of it, both creating and retrieving a config involves only these 2 fields, so calling it `Config` would be preferable.